pkg/k8s/test/integration/cluster_summary_test.go

```diff
@@ -0,0 +1,60 @@
+package integration
+
+import (
+	"bytes"
+	"encoding/json"
+	"testing"
+
+	"github.com/aquasecurity/trivy/pkg/commands"
+	"github.com/aquasecurity/trivy/pkg/flag"
+	"github.com/aquasecurity/trivy/pkg/k8s/report"
+	"github.com/stretchr/testify/assert"
+)
+
+// Note: the test required k8s (kind) cluster installed
+
+func Test_ExecuteK8sClusterScanVulns(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+	actual := new(bytes.Buffer)
```

Comment:
You can export the result to a file and read the file.
https://github.com/aquasecurity/trivy/blob/48e021ea6bf21125d92cb3562b74db121972e17a/integration/module_test.go#L53-L57